setup.py

```diff
@@ -24,8 +24,9 @@
 adwords = ['googleads==12.2.0']
 s3 = ['boto3==1.4.3']
 sftp = ['paramiko==2.0.1']
+rtbhouse = ['rtbhouse_sdk==3.0.1']
 
-all_reports = excel + postgres + drive + adwords + s3 + sftp
+all_reports = excel + postgres + drive + adwords + s3 + sftp + rtbhouse
```

Comment:
What about Rakuten report? Shouldn't it be added as well?

Reply:
rakuten does not have extra dependencies, the only one is requests which is already laika's requirement.